pages/common/mmdc.md

```diff
@@ -0,0 +1,16 @@
+# mermaid-cli
```

Comment:
This needs the match the name of the command being documented, due to the way the infrastructure behind the scenes works.
```suggestion
# mmdc
```